### PR TITLE
build(web): migrate emotion for all components

### PR DESCRIPTION
**Pull request by ShahAnuj2610**
## What this PR does?
migrate emotion to latest version and make code changes accordingly

## Components affected
All web components

## How to test?
checkout this branch and run
```
yarn && yarn dev:web
```
Then run `yarn storybook` inside `packages/playground`

### CODE REVIEW

File: packages/web/package.json [hunk 1]
```diff
@@ -33,16 +33,18 @@
...
-		"emotion": "^9.0.0",
-		"emotion-theming": "^9.0.0",
+		"emotion": "^10.0.27",
+		"emotion-theming": "^10.0.27",
 		"hoist-non-react-statics": "^3.2.1",
 		"polished": "^1.9.3",
 		"prop-types": "^15.6.0",
 		"react-day-picker": "^7.0.5",
-		"react-emotion": "^9.0.0",
+		"react-emotion": "^10.0.0",
```
Comment (jyash97) [inline packages/web/package.json]: Can we remove this ?

Comment (ShahAnuj2610) [reply packages/web/package.json]: yes

File: packages/web/package.json [hunk 2]
```diff
@@ -33,16 +33,17 @@
...
 	"dependencies": {
 		"@appbaseio/reactivecore": "9.6.4",
+		"@emotion/core": "^10.0.28",
+		"@emotion/styled": "^10.0.27",
 		"appbase-js": "^4.1.0",
 		"cross-env": "^5.2.0",
 		"downshift": "^1.31.2",
-		"emotion": "^9.0.0",
-		"emotion-theming": "^9.0.0",
+		"emotion": "^10.0.27",
```
Comment (jyash97) [inline packages/web/package.json]: @ShahAnuj2610  We can remove this as well.

File: packages/maps/src/components/result/ReactiveOpenStreetMap.js [hunk 3]
```diff
@@ -247,7 +249,6 @@ ReactiveOpenStreetMap.propTypes = {
 	defaultPin: types.string,
 	innerClass: types.style,
 	markers: types.children,
-	showMarkers: types.bool,
```
Comment (ShahAnuj2610) [inline packages/maps/src/components/result/ReactiveOpenStreetMap.js:250]: already defined here: https://github.com/appbaseio/reactivesearch/blob/d907282c604b4cc585c47fa7b6b230dd24efe5c3/packages/maps/src/components/result/ReactiveOpenStreetMap.js#L265

**Review by jyash97 — APPROVED** [review]
Works perfectly 🚀

**Review by bietkul — APPROVED** [review]

File: yarn.lock [hunk 4]
```diff
@@ -6647,7 +6647,7 @@ babel-plugin-emotion@^8.0.10:
     source-map "^0.5.7"
     touch "^1.0.0"
 
-babel-plugin-emotion@^9.0.0, babel-plugin-emotion@^9.0.1, babel-plugin-emotion@^9.2.11:
+babel-plugin-emotion@^9.0.0, babel-plugin-emotion@^9.2.11:
```
Comment (nikhil-varma) [inline yarn.lock:6650]: We updated this plugin to `10.0.33`... why does the lock file still show this?

**Comment by nikhil-varma** [comment]
One question, have we checked backwards compatibility?

File: yarn.lock (hunk 4 above)
Comment (nikhil-varma) [reply yarn.lock:6650]: It should be this right?
Screenshot 2020-05-26 at 7 15 35 PM

**Pull request merged**